_source/_posts/2023-07-31-workflows-workshop.md

```diff
@@ -0,0 +1,939 @@
+  
+  _Option 2: Download the application_
+  1. Go to the [okta-enterprise-ready-workshops](https://github.com/oktadev/okta-enterprise-ready-workshops/tree/oidc-workshop-complete) GitHub page
+  2. Open the menu for the green **<> Code** button and select **Download Zip**
+
+### Installing application dependencies
+
+1. Navigate to the folder where you cloned or downloaded the Todo application
+    * If you downloaded the zip file, unzip the application file
+2. Run the following commands to install application dependencies:
+
+```
+npm ci
+npm install passport-http-bearer
+npm install @types/passport-http-bearer -D
+```
+
+### Creating the database
+
+Run the following command to generate a database and seed it with two users.  The user's name and password will write to the console.
+
+```
+npm run init-db
+```
+
+The users are 
+- User: _trinity@whiterabbit.fake_, password: _Zion_
+- User: _bob@tables.fake_, password: _correct horse battery staple_
+
+
+
+### Adding an organziation to the application
+
+In a later step, you will enhance the application with a new API service to get all the todo items. The new API retrieves all the todo items belonging to an organization. 
+
+In this step, you will set up an organization in the application database. 
+
+To launch the database viewer: 
+
+1. In a new terminal tab or window, run the following command from the application folder:
+
+```
+npx prisma studio
+```
+
+You will see the following message
+
+```
+Prisma Studio is up on http://localhost:5555
+```
+
+In a new browser window, go to http://localhost:5555 to view the database. The screenshot below shows the application database. The database has two users. 
+
+{% img blog/workflows-workshop/Workflows_prisma_db.jpg alt:"Application database" %}{: .center-image }
+
+To create a new organiziation in the application database: 
+
+1. Click on the **Org** model
+2. Click **Add record** to add a new record
+3. In the **domain** column, enter **matrix.fake**
+4. In the **apikey** column, enter **131313** (you might need to scroll to the right to see this column)
+5. In the **User** column, click on **0 User** and select both users to add to this organziation (click outside the list to close it)
+5. Click **Save 1 change** button to save the new organization and its API key
+
+The organziation record should look like this (some columns are filtered/hidden to show domain, API, and User columns)
+
+{% img blog/workflows-workshop/Workflows_prisma_db_org_record.jpg alt:"Database with new organization" %}{: .center-image }
+
+
+You are ready to start the application and create the todo items. 
+
+### Running the application
+
+To start the Todo application, run the following command from the application folder: 
+
+```
+npm start
+```
+
+The create serveral todo items:
+
+1. In a browser, go to http://localhost:3000
+2. Sign in to the application. Uuse one of the following users:
+    - User: _trinity@whiterabbit.fake_, password: _Zion_
```

Comment:
Same comment here about adding the users and passwords directly here vs asking people to look at console.

Reply:
Updated.